content/docs/guides/contribute/review-process.en.md

```diff
@@ -0,0 +1,20 @@
+Here's a suggested workflow. The quotation marks indicate the **necessity** of communication to ensure the smooth flow of PR validation:
+
+```mermaid
+sequenceDiagram
+  actor A as PR author
+  actor R as Reviewer/Maintainer
+  
+  A->>R: ASKS "I need a review"
+  R->>A: ANSWERS ✅ "Ok" or ❌ "not ok"
+  
+  R-->>+A: COMMENTS, asks for changments
+  A-->>R: ANSWERS to comments or requests changes
+  A-->>-R: DOES necessary changes in dedicated "fixups"
+  R-->>A: REVIEWS & CLOSES request changes and conversations
+ 
+  A->>R: DOES rebase & fixup
+  R->>A: CHECKS commits
+  R->>A: SAYS ✅ "it's approved"
```

Comment:
The process may not complete in a single review round, especially with first time contributions: there has to be a review loop